scripts/ci/docker-compose/integration-mssql.yml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: "3.8"
+services:
+  mssql:
+    container_name: mssql
+    image: "mcr.microsoft.com/mssql/server:2017-latest"
```

Comment:
I chose this version arbitrarily - should I replace it with the latest version / create a test matrix for all versions?

Reply:
Overkill. We should possibly just bump it to some later version (2017 is really old) - likely 2019 should be good and supported for few more years. We have no promises on what version of mssql we support.